lib/postgrex_wal/messages/relation/column.ex

```diff
@@ -0,0 +1,51 @@
+defmodule PostgrexWal.Messages.Relation.Column do
+  @moduledoc """
+  columns message.
+
+  Int8
+  Flags for the column. Currently can be either 0 for no flags or 1 which marks the column as part of the key.
```

Comment:
A published table must have a “replica identity” configured in order to be able to replicate UPDATE and DELETE operations, so that appropriate rows to update or delete can be identified on the subscriber side. 

By default, this is the primary key, if there is one. Another unique index (with certain additional requirements) can also be set to be the replica identity. If the table does not have any suitable key, then it can be set to replica identity “full”, which means the entire row becomes the key.